daemon/src/projection.rs

```diff
@@ -828,26 +829,70 @@ impl Tx {
         let _ = self.quote.btc_usd.send(quote.map(|q| q.into()));
     }
 
-    fn send_order_update(&self, symbol: ContractSymbol, offer: MakerOffers) {
-        match symbol {
-            ContractSymbol::BtcUsd => {
-                self.order.btc_usd.send(offer).unwrap_or_default();
-            }
-            ContractSymbol::EthUsd => {
-                self.order.eth_usd.send(offer).unwrap_or_default();
-            }
-        }
+    fn send_offer_update(&self, offers: &Offers) -> Result<()> {
+        let OffersByContractSymbol { btc_usd, eth_usd } = split_offers_by_contract_symbol(offers);
+
+        self.order.btc_usd.send(btc_usd)?;
```

Comment:
🙃 `order` should actually be renamed to `offer` - this might be because this is based on an old branch?

Reply:
Nope this is the same in master AFAIK.

Reply:
I think I missed that one when cleaning up 😅 - we can do it in a follow up :)